states/instance_object_src.go

```diff
@@ -49,6 +49,10 @@ type ResourceInstanceObjectSrc struct {
 	// the recommendations in the AttrsJSON documentation above.
 	AttrsFlat map[string]string
 
+	// AttrPaths is an array of paths to mark as sensitive coming out of
```

Comment:
I've had concerns before that the variable and field names used for sensitivity aren't sufficiently specific - Do you think this is going to be extended to save other information, or might a more specific name work here? Something like `AttrSensitivePaths` (or whatever actually makes sense)? 

(So I'm not just making random assertions: This came up in an earlier PR when asking about `marks` - I was confused by the generic name and thought it could refer to something other than  "marked `sensitive`")

Reply:
Sorry, let me also add "and you explained "marks" to me in the PR and your use makes sense, I was the confused one then"

Reply:
Since we're only marking as sensitive, and the data stored in it doesn't include information about what specific mark is used (which we could add, but would be a "breaking" change for this in the future, even if it's additive), I am seeing your point here and am into it, will revisit these names.